tekton/cronjobs/kubectl-image-nightly-build-cron/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: image-build-cron-trigger
+spec:
+  schedule: "0 2 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: trigger
+            env:
+            - name: SINK_URL
+              value: el-image-builder.default.svc.cluster.local:8080
```

Comment:
where does `el-image-builder` come from?

Reply:
https://github.com/tektoncd/plumbing/blob/master/tekton/resources/image-build-trigger.yaml#L19

I should ask the triggers team to drop the 'el-' prefix, it sounds like weird Spanish and I'm not convinced it's really needed for anything :P

Reply:
it sounds like weird Spanish <-- THAT IS EXACTLY WHAT I THOUGHT IT WAS :rofl: